src/home/NotificationBell.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { fireEvent, render } from '@testing-library/react-native'
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import NotificationBell from 'src/home/NotificationBell'
+import { createMockStore } from 'test/utils'
+
+jest.mock('src/analytics/ValoraAnalytics')
+
+const testId = 'NotificationBell'
+
+describe('NotificationBell', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it(`emits the analytics event on press when there is no new notification`, () => {
+    const storeDataWithoutNotification = {
+      account: {
+        backupCompleted: true,
+        dismissedGetVerified: true,
+        celoEducationCompleted: true,
+        dismissedStartSupercharging: true,
+      },
+    }
+
+    const { queryByTestId, getByTestId } = render(
+      <Provider store={createMockStore(storeDataWithoutNotification)}>
+        <NotificationBell testID={testId} />
+      </Provider>
+    )
+
+    expect(queryByTestId(testId)).toBeTruthy()
+    fireEvent.press(getByTestId(testId))
+    expect(ValoraAnalytics.track).toHaveBeenCalledTimes(1)
+    expect(ValoraAnalytics.track).toHaveBeenCalledWith(HomeEvents.notification_bell_pressed, {
+      hasNotifications: false,
+    })
+  })
+
+  it(`emits the analytics event on press when there are notifications`, () => {
+    const { queryByTestId, getByTestId } = render(
+      <Provider store={createMockStore({})}>
+        <NotificationBell testID={testId} />
+      </Provider>
+    )
+
+    expect(queryByTestId(testId)).toBeTruthy()
```

Comment:
nit: for positive assertions,[ we should use the `getBy*` methods ](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence)